dbt/adapters/snowflake/connections.py

```diff
@@ -34,7 +34,7 @@
     RuntimeException,
     FailedToConnectException,
     DatabaseException,
-    warn_or_error,
+    warn_or_error, # type: ignore[attr-defined]
```

Comment:
Is this something new needed?

Reply:
no this will be removed, looking into the failure cause think its around some structure logging changes will fix